docs/modules/data-structures/pages/vector-collections.adoc

```diff
@@ -0,0 +1,494 @@
+= Vector Collection
+:page-enterprise: true
+:page-beta: true
+
+== Intro
+
+The primary object for interacting with the vector storage is a Vector Collection. This collection holds information about the vectors and associated metadata (user value).
+
+A collection consists of one or more indexes, all sharing a common metadata storage. Each index represents a distinct vector space. Vectors from different indexes are stored independently, and indexes can have different configurations.
+
+Conceptually, a vector collection resembles a key-value store. Here, the key is a user-defined unique identifier, and the value is an object containing metadata (user value) alongside several vectors — one for each index.
+
+Users are permitted to store any necessary value for work as metadata (user value). This can be additional characteristics of the source data or even the source data itself.
+
+== Configuration
+
+Collection configuration can be set dynamically during vector collection creation or statically during cluster configuration. Unlike other data structures, the configuration must be set up before the collection can be used. Below is a table listing all available options:
+
+.Collection configuration options
+[cols="1,2,1,1",options="header"]
+|===
+|Option|Description|Required|Default
+
+|name
+|The name of the vector collection.
+|`Required`
```

Comment:
rendering as code or ordinary text is inconsistent between different tables. We need to make it consistent (maybe there are some guidelines?): option/hint/method names, required/optional, default values

Reply:
I think required/optional is rather a text than code, but lets check some guidelines or existing pages

Reply:
653f346d2711640a47b0755a7aa6fbf62f4b5d09

looks like different documents used different formatting. 
At this moment only NULL, Numbers, TRUE\FALSE formatted as code.

Hope that @rebekah-lawrence  will help with the correct formatting

Reply:
Until the style guide is agreed and applied throughout, I'm afraid these inconsistencies will remain. I wouldn't worry about it at the moment. Anything that I suggest would still be inconsistent!

I would - personally - render the Option column entries as code; but again, this is not done consistently through the docs, so it is up to you whether to make that change.